File: James Santoya/Generics/Generics/src/generics/Order.java
package generics;

class Order {

    public String Quantity;
    public String Product_Description;

    public Order() {
    }

    public Order(String Quantity, String Product_Description) {
        this.Quantity = Quantity;
        this.Product_Description = Product_Description;
    }

    public void RetornarDatos() {
        System.out.println("Cantidad : " + Quantity + "   Producto : " + Product_Description);
    }

}

File: YessicaG/Generics/Generics/Order.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.IO;
using System.Windows.Forms;
namespace Generics
{
    public class Order
    {
        public short Quantity { get; set; }

        public string ProductDescription { get; set; }

        public List<Order> GetOrders(string folderName)
        {
            var file = OpenFile(folderName, "orders.txt");
            var ordersList = new List<Order>();
            while (!file.EndOfStream)
            {
                string line = file.ReadLine();
                string[] fields = line.Split(',');
                
                ordersList.Add(new Order()
                {
                    Quantity = short.Parse(fields[0]),
                    ProductDescription = fields[1]
                });
            }
            return ordersList;
        }

             

        private StreamReader OpenFile(string folderName, string fileName)
        {
            return File.OpenText(folderName + "\\" + fileName);
        }
    }
}
